.github/workflows/preview-site.yml

```
@@ -13,15 +13,14 @@ jobs:
       CI: true
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@main
+        uses: actions/checkout@v3
 
       - name: Set up pnpm
-        uses: pnpm/action-setup@v2.2.2
+        uses: pnpm/action-setup@v2
 
-      - name: Set up Node.js 16.x
-        uses: actions/setup-node@main
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
         with:
-          node-version: 16.x
           cache: 'pnpm'
```

We still need to [specify `node-version-file`](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) otherwise it will use the default, which is [16.18.1](https://github.com/seek-oss/braid-design-system/actions/runs/3544167382/jobs/5951211572#step:4:2) and we want [16.13.0](https://github.com/seek-oss/braid-design-system/blob/action-versions/package.json#L77-L79).

```suggestion
          cache: 'pnpm'
          node-version-file: package.json
```

---

Oh yes, you're right.